cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -205,3 +209,23 @@ def _build_spatialized_model_predictions(
             "prediction": aligned_predictions,
         }
     )
+
+
+def _convert_to_h3(df: pd.DataFrame) -> pd.Series:
+    """Builds a Series of h3 indices to the aggregated prediction.
+
+    Args:
+        df: A pandas DataFrame containing the results of
+        _build_spatialized_model_predictions.
+    Returns:
+        A Series of h3 indices to the aggregated prediction for the lat/lon center
+        points which fall within each h3 cell.
+    """
+    df["h3"] = df.apply(_lat_lng_to_h3, axis=1)
+    # TODO: Produce different aggregation(s) depending on prediction type.
```

Comment:
I think we'll actually always need to do `max` here, because we're interested in max flood height & max temperatures per area.